cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/CommandUtil.java

```diff
@@ -134,7 +134,7 @@ public static void initPackageByTemplate(Path path, String packageName, String t
                     StandardCopyOption.REPLACE_EXISTING);
 
             String packageMd = FileUtils.readFileAsString(
-                    NEW_CMD_DEFAULTS + "/" + ProjectConstants.PACKAGE_MD_FILE_NAME);
+                    NEW_CMD_DEFAULTS + "/" + ProjectConstants.PACKAGE_MD_FILE_NAME) + "\n";
```

Comment:
Shouldn't we actually modify the `FileUtils.readFileAsString()` to check the EOF and append a newline character accordingly? Then we can avoid these duplications and it will also make the usages of this util future-proof.